### PR TITLE
Release 3.3.1

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.3.1 (2018-11-29)
+
+- Fixed select tags to display invalid states on touched [#38](https://github.com/blackbaud/skyux-theme/pull/38)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: What about... "Fixed native HTML select fields to use the appropriate CSS style when control value is invalid."

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Fixed selected tab mixin to have correct hover color when used with normal tab mixin [#41](https://github.com/blackbaud/skyux-theme/pull/41)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: "Fixed `sky-btn-tab` SCSS mixin to properly style a tab that is both selected and disabled."

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Wouldn't it be

""Fixed sky-btn-tab SCSS mixin to properly style a tab that is both selected and hovered."?

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: I've updated the text to this. Let me know if that isn't what you want

Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: Hm, I saw the `:not(.sky-btn-tab-disabled)` in the mixin. How about...
"Fixed `sky-btn-tab` SCSS mixin to properly style the hover state for a selected, non-disabled tab."
(Make sure to remember the period and the backticks for the mixin name.)

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

**Pull request merged**